src/OrchardCore.Themes/TheAdmin/Views/Layout.cshtml

```diff
@@ -8,7 +14,11 @@
     <link type="image/x-icon" rel="shortcut icon" href="~/TheAdmin/favicon.ico" />
 
     <!-- Bootstrap CSS -->
-    <style asp-name="admin" asp-src="~/TheAdmin/Styles/TheAdmin.min.css" debug-src="~/TheAdmin/Styles/TheAdmin.css" depends-on="jQuery-ui"></style>
+    <style asp-id="admin-default" asp-media="all" asp-name="admin" asp-src="~/TheAdmin/Styles/TheAdmin.min.css" debug-src="~/TheAdmin/Styles/TheAdmin.css" depends-on="jQuery-ui"></style>
+    @if (adminSettings.DisplayDarkmode)
```

Comment:
I don't like this. I tried using a @section with a Partial view but seems we can't do that.
Any alternative you guys might think of?